lib/pfconfig/namespaces/resource/UriFilters.pm

```diff
@@ -1,44 +1,33 @@
-package pfconfig::namespaces::resource::Profile_Filters;
-
+package pfconfig::namespaces::resource::UriFilters;
 =head1 NAME
 
-pfconfig::namespaces::resource::Profile_Filters
+pfconfig::namespaces::resource::UriFilters
```

Comment:
Well if I take the most recent one (PKI providers), it's been PKI_Provider

But I think it should be URIFilters as I see it as
UniformResourceIdentifierFilters